proto/pocket/service/relay.proto

```diff
@@ -0,0 +1,87 @@
+syntax = "proto3";
+package pocket.service;
+
+option go_package = "pocket/x/service/types";
+
+
+import "cosmos_proto/cosmos.proto";
+import "pocket/service/service.proto";
+import "pocket/application/application.proto";
+import "pocket/supplier/supplier.proto";
+
+// Relay is a protobuf containing both the RelayRequest, signed by the Application
+// and the RelayResponse, signed by the Supplier. Together, the serialized tuple is
+// what is inserted into the SMST leaves as values in the Claim/Proof lifecycle.
+message Relay {
+    RelayRequest req = 1;
+    RelayResponse res = 2;
+}
+
+// RelayMeta is used to hold the metadata for a RelayRequest.
+message RelayRequestMetadata {
+    session.SessionHeader session_header = 1; // The session header for the relay
+    map<string, string> headers = 2; // The headers of the request
+    bytes application_signature = 3; // The signature of the application of the request
```

Comment:
This could also be a `gateway_signature`. Maybe `request_signature` or just `signature` would better reflect its nature.